vars/runPipeline.groovy

```diff
@@ -146,17 +146,19 @@ def testDownstreamProject (name) {
                     // set HOME to separate concurrent ~/.dub user paths
                     "HOME=${env.WORKSPACE}"
                 ]) {
-            try { dir(repo) {
+            try { dir(name) {
 
-                if (repo == 'rejectedsoftware/vibe.d') {
-                    clone("https://github.com/${repo}.git", 'v0.8.3-alpha.1')
+                if (repo == 'vibe-d/vibe.d') {
+                    // https://github.com/vibe-d/vibe.d/pull/2016
+                    // clone("https://github.com/${repo}.git", 'v0.8.3-alpha.1')
+                    clone('https://github.com/MartinNowak/vibe.d.git', 'parallel_tests')
```

Comment:
You can use upstream now, e.g. by cloning commit `5766e5135b77ab62da21e29ed0660ac9178ea093`

Reply:
Clone commit doesn't work.